controllers/tasks.js

```diff
@@ -312,7 +312,7 @@ const updateTaskStatus = async (req, res, next) => {
   try {
     let userStatusUpdate;
     const taskId = req.params.id;
-    const { dev } = req.query;
+    const { userStatusFlag } = req.query;
```

Comment:
Can you please tell what `userStatusFlag` variable means? (not a change request)

Reply:
Why can't we use dev flag instead as these changes have nothing to do with `userStatus`?

Reply:
Hi @Ajeyakrishna-k thanks for  review and I also thinking same way
Renaming it to 'dev' would require widespread code modifications and could disrupt existing functionality from many webesite(my-site, status site) because userStatus used in many places.

Can i handle this in my next pr